app/waf/src/controllers/rule.controller.ts

```diff
@@ -139,4 +147,239 @@ export class RuleController {
   async deleteById(@param.path.string('id') id: string): Promise<void> {
     await this.ruleRepository.deleteById(id);
   }
+  /*Move the conditions and actions related functions here.*/
```

Comment:
So you have done it... Suppose you don't need to say it except you want to say some reason.

Reply:
Removed it

Reply:
removed it